site/content/docs/howto/cicd.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Using Pinniped for CI/CD cluster operations
+description: Using Pinniped for CI/CD cluster operations.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Use Pinniped for CI/CD
+    weight: 500
+    parent: howtos
+---
+
+This guide shows you how to configure Pinniped and your CI/CD system of choice to use Pinniped to administrate Kubernetes clusters.
+
+Note that the guide below assumes that you are using a non-human account within the IDP of your choice.
+It is never recommended to use a human's credentials in this way.
+
+## Prerequisites
+
+This how-to guide assumes that you have already configured the following Pinniped server-side components within your Kubernetes cluster(s):
+
+1. Pinniped Supervisor with a working FederationDomain and IdentityProvider (LDAP, AD, or OIDC)
+   1. Note that the Supervisor installation could be on a completely separate cluster unrelated to your CI/CD
+2. Pinniped Concierge on each cluster that needs to be administrated by your CI/CD system.
```

Comment:
Technically you don't need the Concierge, since you could have configured the API server's OIDC settings (https://pinniped.dev/docs/tutorials/supervisor-without-concierge-demo/). Perhaps this could be worded in such a way that it allows for either?

Reply:
True, but I felt that case didn't need to be addressed here. If they're not using Concierge, but have the API server set up to respect OIDC tokens, they probably don't need any specific CI/CD guidance on that. They could either use their upstream OIDC provider directly, or use Supervisor (or Dex, or UAA) to translate LDAP/AD/SAML into OIDC. 

I think the use-case of this doc is to point out that (A) they need a non-human account in their IDP and (B) they can use `PINNIPED_USERNAME/PASSWORD` for non-interactive logins.

Reply:
But I'll call it out and link to https://pinniped.dev/docs/tutorials/supervisor-without-concierge-demo/, should be pretty simple